aiida-registry-app/src/App.jsx

```diff
@@ -8,15 +8,20 @@ import MaX from './assets/MaX.png'
 import './App.css'
 import jsonData from './plugins_metadata.json'
 import base64Icon from './base64Icon';
-
+import Box from '@mui/material/Box';
+import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import FormControl from '@mui/material/FormControl';
+import Select from '@mui/material/Select';
 
 
 const entrypointtypes = jsonData["entrypointtypes"]
 const globalsummary = jsonData["globalsummary"]
 const plugins  = jsonData["plugins"]
 const status_dict = jsonData["status_dict"]
 const length = Object.keys(plugins).length;
-const currentPath = window.location.pathname;
+const currentPath = import.meta.env.VITE_BASE_PATH || "/aiida-registry/";
```

Comment:
Ah, I see, the `VITE_BASE_PATH` is used here. Can you add a comment on the CI action above about it usage here?

Reply:
I noticed that it shouldn't be in the CI action at all. As it tests the webpage build, not the PR preview.
I renamed the environment variable to `VITE_PR_PREVIEW_PATH`. I think this is more descriptive.